src/arkit.cs

```diff
@@ -2086,4 +2202,103 @@ interface ARMeshGeometry : NSSecureCoding {
 		[NullAllowed]
 		ARGeometrySource Classification { get; }
 	}
+
+	[iOS (14, 0)]
+	[BaseType (typeof (NSObject))]
+	interface ARDepthData {
+		[Export ("depthMap", ArgumentSemantic.Assign)]
+		CVPixelBuffer DepthMap { get; }
+
+		[NullAllowed, Export ("confidenceMap", ArgumentSemantic.Assign)]
+		CVPixelBuffer ConfidenceMap { get; }
+	}
+
+	[iOS (14, 0)]
+	[BaseType (typeof (ARAnchor))]
+	interface ARGeoAnchor : ARTrackable {
+		// Inlined from 'ARAnchorCopying' protocol (we can't have constructors in interfaces)
+		[iOS (14,0)]
+		[Export ("initWithAnchor:")]
+		IntPtr Constructor (ARAnchor anchor);
+
+		[Export ("coordinate")]
+		CLLocationCoordinate2D Coordinate { get; }
+
+		[Export ("altitude")]
+		double Altitude { get; }
+
+		[Export ("altitudeSource", ArgumentSemantic.Assign)]
+		ARAltitudeSource AltitudeSource { get; }
+
+		[Export ("initWithCoordinate:")]
+		IntPtr Constructor (CLLocationCoordinate2D coordinate);
+
+		[Export ("initWithCoordinate:altitude:")]
+		IntPtr Constructor (CLLocationCoordinate2D coordinate, double altitude);
+
+		[Export ("initWithName:coordinate:")]
+		IntPtr Constructor (string name, CLLocationCoordinate2D coordinate);
+
+		[Export ("initWithName:coordinate:altitude:")]
+		IntPtr Constructor (string name, CLLocationCoordinate2D coordinate, double altitude);
+	}
+
+	[iOS (14,0)]
+	[BaseType (typeof (ARConfiguration))]
+	interface ARGeoTrackingConfiguration {
+		[Static]
+		[Export ("supportedVideoFormats")]
+		ARVideoFormat[] GetSupportedVideoFormats ();
+
+		[Export ("environmentTexturing", ArgumentSemantic.Assign)]
+		AREnvironmentTexturing EnvironmentTexturing { get; set; }
+
+		[Export ("wantsHDREnvironmentTextures")]
+		bool WantsHDREnvironmentTextures { get; set; }
```

Comment:
We already have this as `Hdr`: https://github.com/xamarin/xamarin-macios/blob/46b6d3787d50c4bf8bbbade171d6025f8edea602/src/arkit.cs#L1724-L1725

```suggestion
		bool WantsHdrEnvironmentTextures { get; set; }
```